bestie/proto/test_metrics.proto

```diff
@@ -5,6 +5,42 @@ package bestie.proto;
 
 option go_package = "github.com/enfabrica/enkit/bestie/proto";
 
+// Each metric tag must be supplied in the form of a key/value string pair.
+message KeyValuePair {
```

Comment:
`message Tag` is shorter, and if Tag ever becomes something else, it's not complected with other uses of `KeyValuePair`